app/src/main/java/org/wikipedia/search/PrefixSearchResponse.kt

```diff
@@ -0,0 +1,27 @@
+package org.wikipedia.search
+
+import com.google.gson.annotations.SerializedName
+import org.wikipedia.dataclient.mwapi.MwQueryResponse
+
+class PrefixSearchResponse : MwQueryResponse() {
+    @SerializedName("searchinfo")
+    private val searchInfo: SearchInfo? = null
+    private val search: Search? = null
+
+    fun suggestion(): String? {
+        return searchInfo?.suggestion
+    }
+
+    internal class SearchInfo {
+        val suggestion: String? = null
+
+        @SerializedName("suggestionsnippet")
+        private val snippet: String? = null
```

Comment:
Would it be possible to shift `snippet` and to the top of `val suggestion: String? = null`?